api/api_test.go

```diff
@@ -15,26 +14,36 @@ const (
 	execdbname    = "exec.db.test"
 )
 
-func newAPIAndDockerTest(t *testing.T) (*API, *dockertest.Testing, func()) {
-	dt := dockertest.New()
+type apiTesting struct {
+	*testing.T
+	serviceDB   *database.LevelDBServiceDB
+	executionDB *database.LevelDBExecutionDB
+	cm          *mocks.Container
```

Comment:
can we rename it `container` it's hard to know `cm` in the other tests

Reply:
renamed to `containerMock`